src/main/java/binnie/botany/genetics/FlowerSpecies.java

```diff
@@ -68,7 +56,7 @@ public enum FlowerSpecies implements IAlleleFlowerSpecies {
             EnumFlowerColor.WHITE),
     LILY("lily", "lilium", "auratum", EnumFlowerType.LILY, EnumFlowerColor.PINK, EnumFlowerColor.GOLD),
     YARROW("yarrow", "achillea", "millefolium", EnumFlowerType.YARROW, EnumFlowerColor.YELLOW),
-    PETUNIA("petunia", "petunia", "\u00ef?? atkinsiana", EnumFlowerType.PETUNIA, EnumFlowerColor.MEDIUM_VIOLET_RED,
+    PETUNIA("petunia", "petunia", "ï?? atkinsiana", EnumFlowerType.PETUNIA, EnumFlowerColor.MEDIUM_VIOLET_RED,
```

Comment:
I'm fairly certain that we're not going to switch file encodings to something that doesn't support Unicode characters anytime soon. If and when that happens we'll have much bigger problems, or none at all (i.e. automatic conversion).
As for what's wrong with them: They're unreadable without looking them up. Escape characters are generally for characters impossible to otherwise specify in a string, like newlines, not characters that are fully-supported by the format used.